test/acceptance/features/bindAppToServiceUsingConfigMap.feature

```diff
@@ -76,6 +77,7 @@ Feature: Bind values from a config map referred in backing service resource
                     resource: deployments
             """
         Then Service Binding "cmsa-1" is ready
+        And Secret "cmsa-1" does not have key "BACKEND"
```

Comment:
@pratikjagrut why is this required ?

Reply:
This is the step where we're confirming that the operator is not creating data in secret with just KIND as a key which is what happens in the bug #838

Reply:
And this is really not needed to check - only what counts is what application sees. These are acceptance tests and users should be not aware of such implementations details. Hence, please remove, and if needed, check that application does not sees that variable.